api/index.js

```diff
@@ -33,22 +40,37 @@ app.get("/api/board", (req, res) => {
   // Send the current version
   res.sendVersion({
       version: versionNum++,
-      body: JSON.stringify(board)
+      body: JSON.stringify(boards[url])
   })
 
   if (!req.subscribe) res.end()
 });
 
-// app.post("/api/board", (req, res) => {
-//   for (let i = 1; i < 11; i++) {
-//     for (let j = 1; j < 11; j++) {
-//       board[`${i}_${j}`] = ".";
-//     }
-//   }
-//   res.json({ msg: "success!" });
-// });
+app.post("/api/:url", (req, res) => {
```

Comment:
add a comment above this specifying that this endpoint is for resetting the board. i wrote this endpoint and i still forgot what it was for lol